src/Provider/Nominatim/Nominatim.php

```diff
@@ -128,6 +128,10 @@ public function geocodeQuery(GeocodeQuery $query): Collection
             }
         }
 
+        if ((bool) $query->getData('extratags')) {
+            $url .= '&extratags=1';
+        }
```

Comment:
You can add this parameter **by default**, no need to use `$query->withData('extratags', true)`.

Reply:
I don't know why but adding the parameter changes the results. A lot of the tests fail. I'm not that deep into Nominatim, maybe you know more. 

And I also don't know if there is any performance penalty by adding the tags.

Reply:
I checked the Nominatim search logic. It doesn't make it slower, the vast majority of of time is spend searching for the place, adding the bit of extra information from the same table row is trivial in this case and not an extra step (no extra database query). The response data structure is a slight bit larger of course. It has no side effect on the search or response otherwise.

Reply:
Thanks @mtmail ! 👍 
I'll check the failing tests later this weekend.